crates/uv-client/src/flat_index.rs

```diff
@@ -321,6 +321,13 @@ impl<'a> FlatIndexClient<'a> {
                 index: flat_index.clone(),
             });
         }
+
+        dists.sort_by(|a, b| {
```

Comment:
That seems correct, but it's a behavior change, could you open a separate PR for it?